cli/Valet/PhpFpm.php

```diff
@@ -93,6 +103,11 @@ public function updateConfiguration()
             $contents = preg_replace('/^;?listen\.group = .+$/m', 'listen.group = staff', $contents);
             $contents = preg_replace('/^;?listen\.mode = .+$/m', 'listen.mode = 0777', $contents);
         }
+
+        if ($phpVersion) {
+            $contents = str_replace('valet.sock', $this->fpmSockName($phpVersion), $contents);
+        }
+
         $this->files->put($fpmConfigFile, $contents);
```

Comment:
Does `put` overwrite existing files? If not, could we delete an existing version of this file here so we don't have to in `PhpFpm@useVersion`

Reply:
Put overwrites existing files, so yah, we didn't need to delete the old one.